### ISSUE TITLE
Announcement

### ISSUE THREAD

**Issue by Timong-craft** [Its-Vichy/Rankup#1]
I use the bot not as selfbot but I want to send the advertise in a `Annoucement` Channel and it doens't release the message by itself can u fix that?

**Comment by Timong-craft** [comment]
But it is a good Bot

**Comment by Its-Vichy** [comment]
If you are using the program as bot user, add "Bot " before the token

**Comment by Timong-craft** [comment]
ik

**Comment by Timong-craft** [comment]
I have done that, but It doenst publishes the Message by itself

**Comment by Its-Vichy** [comment]
rip

**Comment by Timong-craft** [comment]
Can you do that?

**Comment by Its-Vichy** [comment]
bot must avec access to the channel idk

**Comment by Timong-craft** [comment]
it has and it does post it but not send it to other servers

**Comment by Its-Vichy** [comment]
you need to put the bot on servers -_-

**Comment by Timong-craft** [comment]
No, I want to send it to other servers via "Announcement" Channel which I mentioned earlier but it only get sendet on the server and not on others
![image](https://user-images.githubusercontent.com/77288029/171953131-7cf0a548-6994-4866-86b3-b2d6993edf29.png)

**Comment by Timong-craft** [comment]
Here you can see it (I'm from Germany and beacuse that the text isn't eglish but ok)

**Comment by Timong-craft** [comment]
Hello?

**Comment by Timong-craft** [comment]
Have you understood my problem?

**Comment by Timong-craft** [comment]
This is not my problem
![image](https://user-images.githubusercontent.com/77288029/176452321-1aa9d9b0-6f03-4791-bb45-2c2a8a283067.png)
